### ISSUE TITLE
Hello, is there any plan to support HIPRTC?

### ISSUE THREAD

**Issue by hi20240217** [oneapi-src/unified-runtime#2331]

**Comment by npmiller** [comment]
No, there's no definite plans at the moment to support HIPRTC, we've had some discussions about it previously and it would be good to have but it's a significant amount of work.
